test/tests/outputs/recover/tests

```diff
@@ -8,8 +8,8 @@
                   'test_recover_dir_cp/0005-mesh.cpr/meta_data_mesh.rd/header '
                   'test_recover_dir_cp/0005-restart-0.rd/data '
                   'test_recover_dir_cp/0005-restart-0.rd/header'
-    delete_output_folders = false
     cli_args = 'Executioner/num_steps=5'
+    recover = false
```

Comment:
Why?

Reply:
When recovery is tested, the test harness appends `Outputs/checkpoint=true` to the command used to run the input. If the input already has a checkpoint block defined in it, recovery testing is basically asking MOOSE to create a second `Checkpoint` object, which is no longer allowed (since everything can be accomplished with one object). Furthermore, if the author of a test puts a checkpoint block in their input, it seems reasonable to assume that they also add a test that recovers/restarts from that checkpoint, making it pointless to run the test in recovery mode.